src/compas_occ/brep/brep.py

```diff
@@ -1245,28 +1261,76 @@ def contours(
         """
         raise NotImplementedError
 
-    def slice(self, plane: compas.geometry.Plane) -> BRepFace:
+    def trimmed(self, plane: Union[compas.geometry.Plane, compas.geometry.Frame]):
+        """Trim a BRep with a plane.
```

Comment:
trimming with a plane is slicing

Reply:
Now I'm a little confused. Isn't this the difference per your comment above?
https://github.com/compas-dev/compas_occ/pull/20#discussion_r1149200242

Trim with plane:
![image](https://user-images.githubusercontent.com/3398309/236683656-d6b80c7d-7650-48c5-bff0-5633229775c4.png)

Slice with plane:
![image](https://user-images.githubusercontent.com/3398309/236683751-9cf0da51-c30f-4938-aef3-cd8dfd1ed97b.png)